pocs/tests/test_codestyle.py

```diff
@@ -5,6 +5,9 @@
 def test_conformance():
     """Test that we conform to PEP-8."""
     style = pycodestyle.StyleGuide(quiet=False, ignore=['E501', 'E402'])
-    style.input_dir('pocs')
-    result = style.check_files()
-    assert result.total_errors == 0, "Found code style errors (and warnings)."
+    dirs_to_check = ['pocs', 'peas']
+
+    for d in dirs_to_check:
+        style.input_dir(d)
+        result = style.check_files()
+        assert result.total_errors == 0, "Found code style errors (and warnings)."
```

Comment:
Does this mean that a pocs style error will prevent a peas error from being reported? I.e. doesn't a failed assert end the test?